includes/discount-functions.php

```diff
@@ -383,15 +408,16 @@ function edd_discount_exists( $code_id ) {
  *
  * @since 1.0
```

Comment:
Can we add a 2nd `@since 2.6.11` tag here, and specify that it's when the `update` attribute was added @sunnyratilal